app/scripts/module/openmrs-rest-services/openmrs-patient-relationship.service.js

```diff
@@ -0,0 +1,42 @@
+(function(){
+'use strict';
+angular.module('openmrs-ngresource.restServices')
+.service('PatientResRelationshipService',PatientResRelationshipService);
+
+PatientResRelationshipService.$inject = ['OpenmrsSettings', '$resource','PatientRelationshipModel'];
+function PatientResRelationshipService(OpenmrsSettings,$resource,PatientRelationshipModel){
+  var service;
+  var currentSession;
+  service = {
+    getResource:getResource,
+    getPatientRelationships: getPatientRelationships
+  };
+  return service;
+  function getResource() {
+    var r = $resource(OpenmrsSettings.getCurrentRestUrlBase().trim() + 'relationship');
+        return r;
+      }
+
+      function getPatientRelationships(params,callback){
```

Comment:
@ellymakuba instead of passing callback it is best practice to return the promise instead. You can do,  
`return $resource.get(path);`   //this code returns a promise. Or you can take callbacks and run them but still return the promise. Of coz if someone doesn't pass a callback then you just return a promise. 

Note: If you might need to take care of a situation where the promise is not resolved. At least try to log the error to console.